R/pstr.R

```diff
@@ -5,6 +5,10 @@
 #'
 #' @param companies A dataframe like [pstr_companies].
 #' @param scenarios A dataframe like [pstr_scenarios].
+#' @param low_threshold A numeric value to segment low and medium reduction
+#'   targets.
+#' @param high_threshold A numeric value to segment medium and high reduction
+#'   targets.
```

Comment:
Well done Kalash!

You may also use this to remove duplication and ensure the definition is always the same.

```r
#' @inheritParams xctr
```

Reply:
@maurolepore I apologise for forgetting to keep the PR in the draft mode. The tests take time to finish and then I shift my focus to other work until tests are not complete. That's the reason why I forget to convert the PR from draft mode. I will not repeat this mistake next time.

The definition of `low_threshold` & `high_threshold` is different in case of pstr, hence I would like to keep it that way, but if you would insist to inherit the definitions from xctr then I would be happy to do so.